### PR TITLE
My bad

### PR DESCRIPTION
**Pull request by tungdo0602**

### CODE REVIEW

**Comment by tungdo0602** [comment]
why it still say i have 19 commit breh :/

File: README.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ The token should be in your Clipboard.
 
 paste this into the Console (CTRL + SHIFT + I) on the login screen (you need to be logged out)
 
-```js
+```js&nbsp;
```
Comment (hxr404) [inline README.md:105]:
```suggestion
```js
```

**Review by hxr404 — CHANGES_REQUESTED** [review]
um what exactly did you do? lol

**Comment by hxr404** [comment]
there we go

**Pull request merged**